src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -3,9 +3,9 @@ set_array("sameAs[]")
 
 # 035 - System Control Number (R) - Subfield: $a (NR)
 do list(path:"035  ", "var":"$i")
-  if all_match("$i.a", "\\(DE-605\\)(.*)")
+  if all_match("$i.a", "^\\(DE-605\\)(.*)$")
```

Comment:
Did you mean to change this to `DE-60[05]`? Otherwise the change to `replace_all()` two lines down doesn't make sense.

Reply:
Nope Culturegraph seems to have no ZDB Links. I will undo this.